src/dhdtOverrideManager.cpp

```
@@ -93,20 +93,37 @@ void hdt::Override::OverrideManager::Deserialize(std::stringstream& data_stream)
 	m_ActorPhysicsFileSwapList.clear();
 	try {
-		while (!data_stream.eof()) {
-			uint32_t actor_formID, override_size = 0;
-			data_stream >> std::hex >> actor_formID >> override_size;
+		std::string line;
+		while (std::getline(data_stream, line)) {
+			if (line.empty())
+				continue;
+
+			std::istringstream header(line);
+			uint32_t actor_formID = 0, override_size = 0;
```

Clang (14.0.6)

[warning] variable 'actor_formID' is not initialized

(cppcoreguidelines-init-variables)